src/constraints/ik_constraint.cpp

```diff
@@ -1,7 +1,11 @@
+
+#define _USE_MATH_DEFINES
+#include <cmath>
+#include <algorithm>
```

Comment:
As specified by MSDN
https://docs.microsoft.com/en-us/cpp/c-runtime-library/math-constants?view=msvc-160

Also, due to some include leak (I assume), I had to move this include to the top, or the issue would still happen.

Reply:
Oh weird, but not a big problem...I try to keep the <> includes last, but it's not a hard rule.